app/services/air_now/import_data.rb

```diff
@@ -18,10 +25,9 @@ def locations_endpoint
     'https://s3-us-west-1.amazonaws.com//files.airnowtech.org/airnow/today/monitoring_site_locations.dat'
   end
 
-  def hourly_data_endpoint
-    current_utc = DateTime.now.new_offset(0) - 1.hour
-    formatted_date = current_utc.strftime('%Y%m%d')
-    formatted_hour = current_utc.strftime('%H')
+  def hourly_data_endpoint(utc_time)
+    formatted_date = utc_time.strftime('%Y%m%d')
+    formatted_hour = utc_time.strftime('%H')
```

Comment:
Is that the same as below?
`formatted_time = utc_time.strftime('%Y%m%d%H')`
If so it would make it easier for me to read :)